### PR TITLE
Enable agent TraceExporter configuration in django and flask

### PR DESCRIPTION
**Pull request by lmolkova**
This change allows to use TraceExporter for ocagent with django and flask.

SERVICE_NAME setting is added with the intention to deprecate ZIPKIN_EXPORTER_SERVICE_NAME: if SERVICE_NAME is not present, we check for ZIPKIN_EXPORTER_SERVICE_NAME

### CODE REVIEW

File: opencensus/trace/ext/flask/flask_middleware.py [hunk 1]
```diff
@@ -237,3 +247,13 @@ def _teardown_request(self, exception):
...
         except Exception:  # pragma: NO COVER
             log.error('Failed to trace request', exc_info=True)
+
+    def _get_service_name(self, params):
+        _service_name = params.get(
+            SERVICE_NAME, None)
+
+        if _service_name is None:
+            _service_name = params.get(
+                ZIPKIN_EXPORTER_SERVICE_NAME, 'my_service')
```
Comment (songy23) [inline opencensus/trace/ext/flask/flask_middleware.py]: ditto

File: opencensus/trace/ext/django/middleware.py [hunk 2]
```diff
@@ -216,3 +224,13 @@ def process_response(self, request, response):
...
         finally:
             return response
+
+    def _get_service_name(self, params):
+        _service_name = params.get(
+            SERVICE_NAME, None)
+
+        if _service_name is None:
+            _service_name = params.get(
+                ZIPKIN_EXPORTER_SERVICE_NAME, 'my_service')
```
Comment (songy23) [inline opencensus/trace/ext/django/middleware.py]: If we want to send Zipkin spans to OC-Agent, they need to be converted to OC span proto first. Is it the case in Django?

If that's not the case, please do not add Zipkin as a service endpoint for now. We plan to add an interceptor to accept Zipkin spans in OC-Agent.

Comment (lmolkova) [reply opencensus/trace/ext/django/middleware.py]: this is not about sending the spans, it's just a django/flask app setting for the service name on the exporter.
There was a setting ZIPKIN_EXPORTER_SERVICE_NAME, but I want to generalize it and rename to SERVICE_NAME

Comment (songy23) [reply opencensus/trace/ext/django/middleware.py]: Got it, thanks for the explaination.

**Pull request merged**